product_setup_page/src/allMacros.js

```diff
@@ -4,7 +4,7 @@ let allMacros = [
         macros: [
             {name: 'cs_device_inventory_hostname_postfixes' },
             {name: 'cs_user_inventory_user_postfixes' },
-            {name: 'cs_user_inventory_exclude_users', description: 'To exclude the specific users, add it inside the brackets by comma separated. (e.g. NOT user IN ("user1", "user2"))' },
+            {name: 'cs_user_inventory_exclude_data', description: 'To exclude the users based on some fields. (e.g. NOT (field1=value1 field2=value2))' },
```

Comment:
Let's have it as a common syntax as a filter.

search *

Update description according to that.